crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -434,6 +435,8 @@ impl_param_set!();
 /// [`Exclusive`]: https://doc.rust-lang.org/nightly/std/sync/struct.Exclusive.html
 pub trait Resource: Send + Sync + 'static {}
 
+impl<T: Send + Sync + 'static> Resource for Arc<T> {}
```

Comment:
```suggestion
impl<T: Resource> Resource for Arc<T> {}
```
Isn't this what you meant to do?

Reply:
Fixed, thanks!